### DISCUSSION TITLE
Can X-Axis label be automatically truncated based on pixel width instead of the number of letters

### DISCUSSION THREAD

**Discussion by jiangdongxiaobaiban** [microsoft/fluentui#32107] [General]
Hi team, 
As you can see, the current implementation of truncation relies on `noOfCharsToTruncate` which depends on the count of letters.
Given that each letter has a different width, it may caused overlap in X-axis label. 
Do we have a plan to truncate or wrap to a new line based on width instead of letter count 
![image](https://github.com/user-attachments/assets/73cf0e6c-9a50-452b-b2b3-7dae7ca6ae87)

**Comment by AtishayMsft** [comment]
You can try setting the enableReflow property to true so that no labels overlap with each other. This property uses the pixel based width instead of number of chars.
Alternatively, you can also try the `wrapXAxisLables` property which automatically wraps the labels on each encounter of a whitespace.
